### ISSUE TITLE
Visualize Distances

### ISSUE THREAD

**Issue by birkealine** [MKyhos/bgd-aid-distribution#23]
visualize distances between buildings and the nearest latrine/health service/etc.

- [x] in the db: create voronois polygons for each amenity type
- [ ] include visualization of the voronois in the frontend (maybe color by population per amenity - or for tubewells: by contamination risk)

**Comment by birkealine** [comment]
to make things easier: maybe just display voronoi diagrams? (got this idea from @MKyhos SQL script)
-> shows for which area, which of the tubewells/... is the closest

**Comment by MKyhos** [comment]
Yes, a simple voronoi partition is possible (for an example, see [here](https://mkyhos.github.io/talk_postgis/#20)). I could try to check this out in the next days. This would boil down to:

- one partition per entities of interest (e.g. tubewells, sanitation, nutrition services)
- Maybe, those partitions could be stored as multi polygons in  a single relation
